src/main/java/org/fundacionjala/trello/pages/list/ListPage.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.trello.pages.list;
+
+import org.fundacionjala.trello.pages.core.PageObject;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class ListPage extends PageObject {
+
+    private static final String NAME_LISTS_TEXT_AREA = "textarea.list-header-name";
+
+    @FindBy(css = NAME_LISTS_TEXT_AREA)
+    private List<WebElement> listNames;
+
+    public ListPage(final WebDriver driver) {
+        super(driver);
+    }
+
+    /**
+     * Indicates if the actual page is displayed.
+     *
+     * @return true if actual page is displayed, else false.
+     */
+    @Override
+    public boolean isDisplayed() {
+        return true;
+    }
+
+    public List<String> getAllListsNames() {
+        return listNames.stream().map(WebElement::getText).collect(Collectors.toList());
```

Comment:
I think this should be handled by the Board, because in the context of one list, it only knows its components and not from the other lists